pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeforwardingrule/regionalcomputeforwardingrule/_generated_object_regionalcomputeforwardingrule.golden.yaml

```diff
@@ -1,28 +1,13 @@
-# Copyright 2024 Google LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
 apiVersion: compute.cnrm.cloud.google.com/v1beta1
 kind: ComputeForwardingRule
 metadata:
   annotations:
     cnrm.cloud.google.com/management-conflict-prevention-policy: none
     cnrm.cloud.google.com/project-id: ${projectId}
-    cnrm.cloud.google.com/state-into-spec: merge
```

Comment:
I think not showing this label is expected (and great). Is the `loadBalancingScheme` a output-only field?

Reply:
It can be provided by user, if not, then TF defaults it to "EXTERNAL"([TF code](https://source.corp.google.com/piper///depot/google3/third_party/golang/terraform_providers/google_beta/google_beta/services/compute/resource_compute_forwarding_rule.go;l=232?q=resource_compute_forwarding_rule.go&sq=package:piper%20file:%2F%2Fdepot%2Fgoogle3%20-file:google3%2Fexperimental))

Reply:
Handled TF default values [here](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/924dd21636081ba611483be4361f74068cf5c80d/pkg/controller/direct/compute/forwardingrule_controller.go#L129)